frontend/public/components/namespace.jsx

```diff
@@ -83,10 +91,44 @@ import {
 import { removeQueryArgument } from './utils/router';
 import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
 
+import NamespaceDropdown from '../../packages/console-app/src/components/namespace/NamespaceDropdown';
+
+import {
+  isCurrentUser,
+  isOtherUser,
+  isSystemNamespace,
+} from '../../packages/console-app/src/components/namespace/filters';
+
 const getModel = (useProjects) => (useProjects ? ProjectModel : NamespaceModel);
 const getDisplayName = (obj) =>
   _.get(obj, ['metadata', 'annotations', 'openshift.io/display-name']);
-const CREATE_NEW_RESOURCE = '#CREATE_RESOURCE_ACTION#';
+
+export const REQUEST_FILTER = { me: 'me', user: 'user', system: 'system' };
+
+const getFilters = [
```

Comment:
@spadgett  brought this up when I commented on this being a callback, but since we are using `i18next.t` below, this should be a callback so that it is reactive. Otherwise, the translations will not change when the user selects a new language.

Reply:
Turned getFilters into a function to better match what is done in pod.tsx.  Verified that these filters are translated when switching languages.